<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgProductScoreClass</class>
 <widget class="QDialog" name="DlgProductScoreClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DlgProductScore</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5,3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>638</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_close">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    padding: 8px 16px 8px 36px; 
    font-size: 14px;
    qproperty-icon: url(:/ButtonScanner/image/exit.png);  
    qproperty-iconSize: 30px 30px;
}

QPushButton:hover {
    background-color: #ff6666;
}

QPushButton:pressed {
    background-color: #cc0000;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: rgb(218, 218, 218);
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="styleSheet">
           <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QRadioButton" name="rbtn_queyaEnable">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton {
    spacing: 8px;
    font-size: 14px;
    color: #333;  /* 保持原始文字颜色 */
}

/* 未选中状态 */
QRadioButton::indicator {
    width: 20px;
    height: 20px;
    border-radius: 10px;
    border: 2px solid #999;
    background: #ffffff;
}

/* 悬停效果 */
QRadioButton::indicator:hover {
    border: 2px solid #666;
}

/* 选中状态 */
QRadioButton::indicator:checked {
    border: 2px solid #FF6B00;  /* 橙色边框 */
}

/* 选中时的内圆 */
QRadioButton::indicator:checked {
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.4,
        fx:0.5, fy:0.5,
        stop:0 #FF6B00,  /* 主橙色 */
        stop:1 transparent
    );
}

/* 禁用状态 */
QRadioButton:::disabled {
    border: 2px solid #ccc;
    background: #eee;
}

/* 点击动画效果 */
QRadioButton::indicator:pressed {
    border: 2px solid #CC5500;  /* 深橙色边框 */
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.5,
        fx:0.5, fy:0.5,
        stop:0 #FFD8B3,  /* 浅橙色渐变 */
        stop:1 transparent
    );
}</string>
             </property>
             <property name="text">
              <string>缺牙</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">/* 基础标签样式 */
QLabel {
    color: #444444;          /* 主文字色 */
    font-size: 14px;         /* 标准字号 */
    background: transparent; /* 透明背景 */
    padding: 2px 4px;        /* 基础内边距 */
}

/* 标题样式 */
QLabel.title {
    font-size: 16px;
    font-weight: 600;
    color: #333333;
    border-bottom: 1px solid #DDDDDD; /* 下划线装饰 */
    padding-bottom: 4px;
}

/* 副标题/次要文本 */
QLabel.subtitle {
    color: #666666;
    font-size: 13px;
    font-style: italic;
}

/* 强调文本 */
QLabel.emphasis {
    color: #333333;
    font-weight: 500;
    background-color: #F5F5F5;
    border-radius: 3px;
    padding: 4px 8px;
}

/* 状态标签（成功/警告/错误） */
QLabel[state=&quot;success&quot;] {
    color: #228B22;          /* 森林绿 */
    border-left: 3px solid #228B22;
    padding-left: 8px;
}

QLabel[state=&quot;warning&quot;] {
    color: #FF8C00;          /* 暗橙 */
    border-left: 3px solid #FF8C00;
    padding-left: 8px;
}

QLabel[state=&quot;error&quot;] {
    color: #B22222;          /* 砖红 */
    border-left: 3px solid #B22222;
    padding-left: 8px;
}

/* 数据展示标签 */
QLabel.data {
    color: #2F4F4F;          /* 深石板灰 */
    font-family: monospace;
    border: 1px solid #E0E0E0;
    background-color: #FFFFFF;
    padding: 6px 12px;
    border-radius: 4px;
}

/* 禁用状态 */
QLabel:disabled {
    color: #AAAAAA;
    background-color: transparent;
}

/* 链接样式（伪链接效果） */
QLabel.link {
    color: #4682B4;          /* 钢蓝 */
    text-decoration: underline;
    border-bottom: 1px dotted #4682B4;
}

/* 图标标签 */
QLabel.icon {
    qproperty-alignment: AlignCenter;
    border: 1px solid #E0E0E0;
    border-radius: 4px;
    background-color: #FFFFFF;
    padding: 12px;
}</string>
             </property>
             <property name="text">
              <string>相似度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ptn_queyaSimilarity">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
             </property>
             <property name="text">
              <string>value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_25">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">/* 基础标签样式 */
QLabel {
    color: #444444;          /* 主文字色 */
    font-size: 14px;         /* 标准字号 */
    background: transparent; /* 透明背景 */
    padding: 2px 4px;        /* 基础内边距 */
}

/* 标题样式 */
QLabel.title {
    font-size: 16px;
    font-weight: 600;
    color: #333333;
    border-bottom: 1px solid #DDDDDD; /* 下划线装饰 */
    padding-bottom: 4px;
}

/* 副标题/次要文本 */
QLabel.subtitle {
    color: #666666;
    font-size: 13px;
    font-style: italic;
}

/* 强调文本 */
QLabel.emphasis {
    color: #333333;
    font-weight: 500;
    background-color: #F5F5F5;
    border-radius: 3px;
    padding: 4px 8px;
}

/* 状态标签（成功/警告/错误） */
QLabel[state=&quot;success&quot;] {
    color: #228B22;          /* 森林绿 */
    border-left: 3px solid #228B22;
    padding-left: 8px;
}

QLabel[state=&quot;warning&quot;] {
    color: #FF8C00;          /* 暗橙 */
    border-left: 3px solid #FF8C00;
    padding-left: 8px;
}

QLabel[state=&quot;error&quot;] {
    color: #B22222;          /* 砖红 */
    border-left: 3px solid #B22222;
    padding-left: 8px;
}

/* 数据展示标签 */
QLabel.data {
    color: #2F4F4F;          /* 深石板灰 */
    font-family: monospace;
    border: 1px solid #E0E0E0;
    background-color: #FFFFFF;
    padding: 6px 12px;
    border-radius: 4px;
}

/* 禁用状态 */
QLabel:disabled {
    color: #AAAAAA;
    background-color: transparent;
}

/* 链接样式（伪链接效果） */
QLabel.link {
    color: #4682B4;          /* 钢蓝 */
    text-decoration: underline;
    border-bottom: 1px dotted #4682B4;
}

/* 图标标签 */
QLabel.icon {
    qproperty-alignment: AlignCenter;
    border: 1px solid #E0E0E0;
    border-radius: 4px;
    background-color: #FFFFFF;
    padding: 12px;
}</string>
             </property>
             <property name="text">
              <string>面积</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ptn_queyaArea">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
             </property>
             <property name="text">
              <string>value</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="styleSheet">
           <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QRadioButton" name="rbtn_tangshangEnable">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton {
    spacing: 8px;
    font-size: 14px;
    color: #333;  /* 保持原始文字颜色 */
}

/* 未选中状态 */
QRadioButton::indicator {
    width: 20px;
    height: 20px;
    border-radius: 10px;
    border: 2px solid #999;
    background: #ffffff;
}

/* 悬停效果 */
QRadioButton::indicator:hover {
    border: 2px solid #666;
}

/* 选中状态 */
QRadioButton::indicator:checked {
    border: 2px solid #FF6B00;  /* 橙色边框 */
}

/* 选中时的内圆 */
QRadioButton::indicator:checked {
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.4,
        fx:0.5, fy:0.5,
        stop:0 #FF6B00,  /* 主橙色 */
        stop:1 transparent
    );
}

/* 禁用状态 */
QRadioButton:::disabled {
    border: 2px solid #ccc;
    background: #eee;
}

/* 点击动画效果 */
QRadioButton::indicator:pressed {
    border: 2px solid #CC5500;  /* 深橙色边框 */
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.5,
        fx:0.5, fy:0.5,
        stop:0 #FFD8B3,  /* 浅橙色渐变 */
        stop:1 transparent
    );
}</string>
             </property>
             <property name="text">
              <string>烫伤</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">/* 基础标签样式 */
QLabel {
    color: #444444;          /* 主文字色 */
    font-size: 14px;         /* 标准字号 */
    background: transparent; /* 透明背景 */
    padding: 2px 4px;        /* 基础内边距 */
}

/* 标题样式 */
QLabel.title {
    font-size: 16px;
    font-weight: 600;
    color: #333333;
    border-bottom: 1px solid #DDDDDD; /* 下划线装饰 */
    padding-bottom: 4px;
}

/* 副标题/次要文本 */
QLabel.subtitle {
    color: #666666;
    font-size: 13px;
    font-style: italic;
}

/* 强调文本 */
QLabel.emphasis {
    color: #333333;
    font-weight: 500;
    background-color: #F5F5F5;
    border-radius: 3px;
    padding: 4px 8px;
}

/* 状态标签（成功/警告/错误） */
QLabel[state=&quot;success&quot;] {
    color: #228B22;          /* 森林绿 */
    border-left: 3px solid #228B22;
    padding-left: 8px;
}

QLabel[state=&quot;warning&quot;] {
    color: #FF8C00;          /* 暗橙 */
    border-left: 3px solid #FF8C00;
    padding-left: 8px;
}

QLabel[state=&quot;error&quot;] {
    color: #B22222;          /* 砖红 */
    border-left: 3px solid #B22222;
    padding-left: 8px;
}

/* 数据展示标签 */
QLabel.data {
    color: #2F4F4F;          /* 深石板灰 */
    font-family: monospace;
    border: 1px solid #E0E0E0;
    background-color: #FFFFFF;
    padding: 6px 12px;
    border-radius: 4px;
}

/* 禁用状态 */
QLabel:disabled {
    color: #AAAAAA;
    background-color: transparent;
}

/* 链接样式（伪链接效果） */
QLabel.link {
    color: #4682B4;          /* 钢蓝 */
    text-decoration: underline;
    border-bottom: 1px dotted #4682B4;
}

/* 图标标签 */
QLabel.icon {
    qproperty-alignment: AlignCenter;
    border: 1px solid #E0E0E0;
    border-radius: 4px;
    background-color: #FFFFFF;
    padding: 12px;
}</string>
             </property>
             <property name="text">
              <string>相似度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbtn_tangshangSimilarity">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
             </property>
             <property name="text">
              <string>value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_27">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">/* 基础标签样式 */
QLabel {
    color: #444444;          /* 主文字色 */
    font-size: 14px;         /* 标准字号 */
    background: transparent; /* 透明背景 */
    padding: 2px 4px;        /* 基础内边距 */
}

/* 标题样式 */
QLabel.title {
    font-size: 16px;
    font-weight: 600;
    color: #333333;
    border-bottom: 1px solid #DDDDDD; /* 下划线装饰 */
    padding-bottom: 4px;
}

/* 副标题/次要文本 */
QLabel.subtitle {
    color: #666666;
    font-size: 13px;
    font-style: italic;
}

/* 强调文本 */
QLabel.emphasis {
    color: #333333;
    font-weight: 500;
    background-color: #F5F5F5;
    border-radius: 3px;
    padding: 4px 8px;
}

/* 状态标签（成功/警告/错误） */
QLabel[state=&quot;success&quot;] {
    color: #228B22;          /* 森林绿 */
    border-left: 3px solid #228B22;
    padding-left: 8px;
}

QLabel[state=&quot;warning&quot;] {
    color: #FF8C00;          /* 暗橙 */
    border-left: 3px solid #FF8C00;
    padding-left: 8px;
}

QLabel[state=&quot;error&quot;] {
    color: #B22222;          /* 砖红 */
    border-left: 3px solid #B22222;
    padding-left: 8px;
}

/* 数据展示标签 */
QLabel.data {
    color: #2F4F4F;          /* 深石板灰 */
    font-family: monospace;
    border: 1px solid #E0E0E0;
    background-color: #FFFFFF;
    padding: 6px 12px;
    border-radius: 4px;
}

/* 禁用状态 */
QLabel:disabled {
    color: #AAAAAA;
    background-color: transparent;
}

/* 链接样式（伪链接效果） */
QLabel.link {
    color: #4682B4;          /* 钢蓝 */
    text-decoration: underline;
    border-bottom: 1px dotted #4682B4;
}

/* 图标标签 */
QLabel.icon {
    qproperty-alignment: AlignCenter;
    border: 1px solid #E0E0E0;
    border-radius: 4px;
    background-color: #FFFFFF;
    padding: 12px;
}</string>
             </property>
             <property name="text">
              <string>面积</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbtn_tangshangArea">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
             </property>
             <property name="text">
              <string>value</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="styleSheet">
           <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QRadioButton" name="rbtn_zangwuEnable">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton {
    spacing: 8px;
    font-size: 14px;
    color: #333;  /* 保持原始文字颜色 */
}

/* 未选中状态 */
QRadioButton::indicator {
    width: 20px;
    height: 20px;
    border-radius: 10px;
    border: 2px solid #999;
    background: #ffffff;
}

/* 悬停效果 */
QRadioButton::indicator:hover {
    border: 2px solid #666;
}

/* 选中状态 */
QRadioButton::indicator:checked {
    border: 2px solid #FF6B00;  /* 橙色边框 */
}

/* 选中时的内圆 */
QRadioButton::indicator:checked {
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.4,
        fx:0.5, fy:0.5,
        stop:0 #FF6B00,  /* 主橙色 */
        stop:1 transparent
    );
}

/* 禁用状态 */
QRadioButton:::disabled {
    border: 2px solid #ccc;
    background: #eee;
}

/* 点击动画效果 */
QRadioButton::indicator:pressed {
    border: 2px solid #CC5500;  /* 深橙色边框 */
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.5,
        fx:0.5, fy:0.5,
        stop:0 #FFD8B3,  /* 浅橙色渐变 */
        stop:1 transparent
    );
}</string>
             </property>
             <property name="text">
              <string>脏污</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">/* 基础标签样式 */
QLabel {
    color: #444444;          /* 主文字色 */
    font-size: 14px;         /* 标准字号 */
    background: transparent; /* 透明背景 */
    padding: 2px 4px;        /* 基础内边距 */
}

/* 标题样式 */
QLabel.title {
    font-size: 16px;
    font-weight: 600;
    color: #333333;
    border-bottom: 1px solid #DDDDDD; /* 下划线装饰 */
    padding-bottom: 4px;
}

/* 副标题/次要文本 */
QLabel.subtitle {
    color: #666666;
    font-size: 13px;
    font-style: italic;
}

/* 强调文本 */
QLabel.emphasis {
    color: #333333;
    font-weight: 500;
    background-color: #F5F5F5;
    border-radius: 3px;
    padding: 4px 8px;
}

/* 状态标签（成功/警告/错误） */
QLabel[state=&quot;success&quot;] {
    color: #228B22;          /* 森林绿 */
    border-left: 3px solid #228B22;
    padding-left: 8px;
}

QLabel[state=&quot;warning&quot;] {
    color: #FF8C00;          /* 暗橙 */
    border-left: 3px solid #FF8C00;
    padding-left: 8px;
}

QLabel[state=&quot;error&quot;] {
    color: #B22222;          /* 砖红 */
    border-left: 3px solid #B22222;
    padding-left: 8px;
}

/* 数据展示标签 */
QLabel.data {
    color: #2F4F4F;          /* 深石板灰 */
    font-family: monospace;
    border: 1px solid #E0E0E0;
    background-color: #FFFFFF;
    padding: 6px 12px;
    border-radius: 4px;
}

/* 禁用状态 */
QLabel:disabled {
    color: #AAAAAA;
    background-color: transparent;
}

/* 链接样式（伪链接效果） */
QLabel.link {
    color: #4682B4;          /* 钢蓝 */
    text-decoration: underline;
    border-bottom: 1px dotted #4682B4;
}

/* 图标标签 */
QLabel.icon {
    qproperty-alignment: AlignCenter;
    border: 1px solid #E0E0E0;
    border-radius: 4px;
    background-color: #FFFFFF;
    padding: 12px;
}</string>
             </property>
             <property name="text">
              <string>相似度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbtn_zangwuSimilarity">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
             </property>
             <property name="text">
              <string>value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_26">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">/* 基础标签样式 */
QLabel {
    color: #444444;          /* 主文字色 */
    font-size: 14px;         /* 标准字号 */
    background: transparent; /* 透明背景 */
    padding: 2px 4px;        /* 基础内边距 */
}

/* 标题样式 */
QLabel.title {
    font-size: 16px;
    font-weight: 600;
    color: #333333;
    border-bottom: 1px solid #DDDDDD; /* 下划线装饰 */
    padding-bottom: 4px;
}

/* 副标题/次要文本 */
QLabel.subtitle {
    color: #666666;
    font-size: 13px;
    font-style: italic;
}

/* 强调文本 */
QLabel.emphasis {
    color: #333333;
    font-weight: 500;
    background-color: #F5F5F5;
    border-radius: 3px;
    padding: 4px 8px;
}

/* 状态标签（成功/警告/错误） */
QLabel[state=&quot;success&quot;] {
    color: #228B22;          /* 森林绿 */
    border-left: 3px solid #228B22;
    padding-left: 8px;
}

QLabel[state=&quot;warning&quot;] {
    color: #FF8C00;          /* 暗橙 */
    border-left: 3px solid #FF8C00;
    padding-left: 8px;
}

QLabel[state=&quot;error&quot;] {
    color: #B22222;          /* 砖红 */
    border-left: 3px solid #B22222;
    padding-left: 8px;
}

/* 数据展示标签 */
QLabel.data {
    color: #2F4F4F;          /* 深石板灰 */
    font-family: monospace;
    border: 1px solid #E0E0E0;
    background-color: #FFFFFF;
    padding: 6px 12px;
    border-radius: 4px;
}

/* 禁用状态 */
QLabel:disabled {
    color: #AAAAAA;
    background-color: transparent;
}

/* 链接样式（伪链接效果） */
QLabel.link {
    color: #4682B4;          /* 钢蓝 */
    text-decoration: underline;
    border-bottom: 1px dotted #4682B4;
}

/* 图标标签 */
QLabel.icon {
    qproperty-alignment: AlignCenter;
    border: 1px solid #E0E0E0;
    border-radius: 4px;
    background-color: #FFFFFF;
    padding: 12px;
}</string>
             </property>
             <property name="text">
              <string>面积</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbtn_zangwuArea">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
             </property>
             <property name="text">
              <string>value</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>328</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
